view/frontend/web/js/view/payment/method-renderer/omise-cc-method.js

```diff
@@ -257,6 +257,24 @@ define(
                             redirectOnSuccessAction.execute();
                         }
                     });
+            },
+
+            getSelectedTokenBillingAddress: function() {
+                var selectedBillingAddress = quote.billingAddress();
+                var address = {
+                    state          : selectedBillingAddress.region,
+                    postal_code: selectedBillingAddress.postcode,
+                    phone_number: selectedBillingAddress.telephone,
+                    country: selectedBillingAddress.countryId,
+                    city: selectedBillingAddress.city
+                }
+                if(selectedBillingAddress.street.length) {
+                    address.street1 = selectedBillingAddress.street[0]
```

Comment:
Do we need to assign these `street*` attributes conditionally?  Would it be bad if we just assigned them as-is in the `address` var above since JavaScript has no issue with "out of bounds" 🤷‍♂️?

Reply:
`selectedBillingAddress.street` here it can be ["street1"] or ["street1", "street2"] which depends on how user fill in billing address. I'll remove first "if" as street address is mandatory but keep second condition to check if it exists.

Reply:
🙏